sources/HUBUtilities.h

```diff
@@ -55,7 +55,7 @@ static inline BOOL HUBPropertyIsEqual(NSObject * _Nullable objectA, NSObject * _
  *
  *  This function asserts that a view has been loaded after -loadView was sent to the component.
  */
-static inline UIView *HUBComponentLoadViewIfNeeded(id<HUBComponent> component) {
+static inline UIView *HUBComponentLoadViewIfNeeded(id<HUBComponent> _Nullable component) {
```

Comment:
Do we need to make this change? (This function is being used in many places, and it doesn't make a lot of sense to call it with `nil`)